djxml/xmlmodels/base.py

```diff
@@ -12,10 +12,14 @@
                                     MultipleObjectsReturned,)
 from django.db.models.base import subclass_exception
 try:
-    from django.utils.encoding import (
-        smart_bytes as smart_str, force_text as force_unicode)
+    from django.utils.encoding import force_text as force_unicode
 except ImportError:
-    from django.utils.encoding import smart_str, force_unicode
+    from django.utils.encoding import force_unicode
```

Comment:
I think it would be better to do this in a `try: except: else:`, e.g.

```python
try:
   from django.utils.encoding import force_text as force_unicode, smart_text, smart_bytes
except ImportError:
   from django.utils.encoding import smart_str, force_unicode
else:
    smart_str = smart_text if six.PY3 else smart_bytes
```

This will need more reworking, since in python 3 they've gone ahead and renamed `smart_text` to `smart_str` and deprecated the former... but we'll cross that bridge when we get there.

Reply:
Yeah, I definitely wouldn't use `force_unicode` since that is deprected.

Reply:
I got a couple questions:

1. There is [only 1 line of code](https://github.com/theatlantic/django-xml/pull/12/files#diff-463b28357be7075ed9238b728cc24d07R238) attempting to use `force_unicode` in this file. I was thinking of change that line to use `force_text`, then have a look at the imports.
2. Should we be using Django's `django.utils.six` instead?
3. I'm trying to get clarity on the goals here for imports adjustments: we want to use `force_text` over `force_unicode`, and import `smart_text`|`smart_str` & `smart_bytes`, redefining `smart_str` equal to the correct Py2 vs Py3 string utility. Does that sound about right?